packages/orchestrator-ui-components/src/components/WFOSubscription/SubscriptionKeyValueBlock.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { EuiThemeComputed } from '@elastic/eui/src/services/theme/types';
+import { EuiFlexGroup, EuiFlexItem, EuiSpacer, EuiText } from '@elastic/eui';
+
+import { WFOCheckmarkCircleFill, WFOMinusCircleOutline } from '../../icons';
+import { WFOSubscriptionStatusBadge } from '../WFOBadges';
+import {
+    WFOKeyValueTable,
+    WFOKeyValueTableDataType,
+} from '../WFOKeyValueTable/WFOKeyValueTable';
+import { useOrchestratorTheme } from '../../hooks';
+
+import { KeyValue, SubscriptionStatus } from '../../types';
+
+export const RenderField = (
```

Comment:
The RenderField is actually a function returning a component / JSX. Therefore I would suggest to rename this into getRenderField or at least not capitalize the name. (not sure if in scope of PR)

Reply:
renamed it to getRenderField